api/src/test/resources/org/openmrs/include/standardTestDataset.xml

```diff
@@ -237,7 +237,7 @@
   <location location_id="3" name="Never Never Land" description="This is never be found" address1="" address2="" city_village="Peter" state_province="Pan" postal_code="" country="Wendy" latitude="" longitude="" creator="1" date_created="2008-08-15 15:28:54.0" retired="true" retired_by="1" date_retired="2008-08-15 00:00:00.0" retire_reason="test" uuid="167ce20c-4785-4285-9119-d197268f7f4a"/>
   <person person_id="1" gender="M" birthdate="1948-01-01 00:00:00.0" birthdate_estimated="0" dead="false" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="ba1b19c2-3ed6-4f63-b8c0-f762dc8d7562"/>
   <person person_id="2" gender="M" birthdate="1975-04-08 00:00:00.0" birthdate_estimated="0" dead="false" creator="1" date_created="2005-09-22 00:00:00.0" voided="false" uuid="da7f524f-27ce-4bb2-86d6-6d1d05312bd5"/>
-  <person person_id="6" gender="M" birthdate="2007-05-27 00:00:00.0" birthdate_estimated="0" dead="false" creator="1" date_created="2006-01-18 00:00:00.0" voided="false" uuid="a7e04421-525f-442f-8138-05b619d16def"/>
+  <person person_id="6" gender="M" birthdate="1975-05-27 00:00:00.0" birthdate_estimated="0" dead="false" creator="1" date_created="2006-01-18 00:00:00.0" voided="false" uuid="a7e04421-525f-442f-8138-05b619d16def"/>
```

Comment:
do we need the change in the dataset now that you create patients/visits from scratch?

Reply:
Yes it's still needed. Without this change the following tests fail because they make a visit in 2005.
mergePatients_shouldChangeUserRecordsOfNonPreferredPersonToPreferredPerson
mergePatients_shouldMergeVisitsFromNonPreferredToPreferredPatient
mergePatients_shouldVoidNonPreferredPersonObject

I could also look at changing that visit's start date but it seemed lower impact to change this patient's birthday to me.

Reply:
its ok then, just wanted to know. thanks!